module/merintr/alias.c

```diff
@@ -85,6 +85,21 @@ static INT_PTR CALLBACK VerbAliasDialogProc(HWND hDlg, UINT message, WPARAM wPar
 extern Bool	gbClassicKeys;
 extern player_info *GetPlayer(void);
 
+/****************************************************************************/
+/*
+ * UpdateKeyTables:  Update either the classic or custom keys interfaces table
+ * depending on the client configuration, then also update the inventory key
+ * table so that the hotkey aliases work when the user is in that child window
+ */
+static void UpdateKeyTables(int key, WORD command, char *text)
```

Comment:
This is minor, but it's better to put helper functions like this closer to the end of the file, so that important functions like "AliasInit" come first, as they're more likely to be what a reader is looking for.

In order to get that to compile, you'll need to forward declare UpdateKeyTables.  Line 84 would be a good spot.

Reply:
Thanks.  I didn't know about forward declarations and that was helpful.  I have done as you suggested.